### PR TITLE
SUS-693 add a warning when chat message exceeds max. length

### PR DESCRIPTION
**Pull request by gitkuba**
[SUS-693](https://wikia-inc.atlassian.net/browse/SUS-693)

When chat message exceeds its max. length, following message will appear:
"Sorry, but your message is too long. You may only send no more than ... characters"

@mixth-sense @sqreek

### CODE REVIEW

File: extensions/wikia/Chat2/Chat.i18n.php [hunk 1]
```diff
@@ -119,7 +119,9 @@
 	'chat-browser-is-notsupported' => '	Your browser is not supported. For the best experience, use a <a href="http://community.wikia.com/wiki/Help:Supported_browsers">newer browser</a>.',
 	'chat-checkuser-join-action' => 'Joined Chat',
 
-	'chat-status-away' => 'Away'
+	'chat-status-away' => 'Away',
+	
+	'chat-limit-reached' => 'Sorry, but your message is too long. You may only send no more than $1 characters',
```
Comment (adamkarminski) [inline extensions/wikia/Chat2/Chat.i18n.php]: Please, add a message documentation under the [qqq key](https://github.com/Wikia/app/blob/f8b5b3d50370428a7830d71835cc127a0d784e60/extensions/wikia/Chat2/Chat.i18n.php#L138-L206)

Comment (ghost) [inline extensions/wikia/Chat2/Chat.i18n.php]: @gitkuba — for more information, refer to https://www.mediawiki.org/wiki/Help:System_message#Finding_messages_and_documentation

File: extensions/wikia/Chat2/js/controllers/controllers.js [hunk 2]
```diff
@@ -300,6 +300,7 @@ var NodeRoomController = $.createClass(Observable, {
 
 		// display character count if nearing limit
 		$('.remaining').text(remaining);
+		$('.limit-reached-msg').text(mw.message('chat-message-was-too-long').escaped());
```
Comment (lukaszkonieczny) [inline extensions/wikia/Chat2/js/controllers/controllers.js]: I think we should check if character limit was already exceeded.

**Comment by ghost** [comment]
Per @lukaszkonieczny — where do we check whether the limit has been reached?

**Comment by gitkuba** [comment]
@mixth-sense  @lukaszkonieczny 
it's right here :
https://github.com/Wikia/app/blob/966e91bb6f62548dfef2005d74ecd221eceb8d3f/extensions/wikia/Chat2/js/controllers/controllers.js#L304-L307
as the limit is reached, 'limit-reached' css class is toggled :)
https://github.com/Wikia/app/blob/966e91bb6f62548dfef2005d74ecd221eceb8d3f/extensions/wikia/Chat2/css/Chat.scss#L41-L43

**Comment by ghost** [comment]
Great, thanks!

**Comment by ghost** [comment]
Tested and accepted!

**Pull request merged**